products/intelprocessors.md

```diff
@@ -0,0 +1,171 @@
+---
+permalink: /intel-processors
+title: Intel Processors
+iconSlug: intel
+category: device
+versionCommand: "'msinfo32' on Windows, 'cat /proc/cpuinfo' on Linux"
+releasePolicyLink: https://www.intel.com/content/www/us/en/support/articles/000022396/processors.html
+discontinuedColumn: true
+activeSupportColumn: false
+eolColumn: Active Support
+latestColumn: false
+releaseColumn: false
+releaseDateColumn: true
+sortReleasesBy: 'releaseDate'
+releases:
+-   releaseCycle: "Ivy Bridge (excl. Xeon E5v2/E7v2)"
```

Comment:
Let's split up releaseCycle and releaseLabels to make for a cleaner API. Unsure about using snake_case, but I think that's how Intel returns results on the CLI - Don't have a 2 word processor to check against.

```yaml
- releaseCycle: ivy_bridge
  releaseLabel: Ivy Bridge

- releaseCycle: ivy_bridge_server
  releaseLabel: Xeon E5v2/E7v2
```
Or alternatively:
```yaml
- releaseCycle: ivy_bridge_server (Or maybe ivy_bridge_xeon)
  releaseLabel: Ivy Bridge (Xeon E5v2/E7v2)
```

Reply:
Did I miss something with https://github.com/endoflife-date/endoflife.date/pull/1397/commits/9b5c3aca20c3f03a4887a41047376566694f2f7d ?

Reply:
Yeah, the remaining releases still need updating - 9b5c3aca20c3f03a4887a41047376566694f2f7d only sets it for 2.

Reply:
> Yeah, the remaining releases still need updating - [9b5c3ac](https://github.com/endoflife-date/endoflife.date/commit/9b5c3aca20c3f03a4887a41047376566694f2f7d) only sets it for 2.

https://github.com/endoflife-date/endoflife.date/pull/1397/commits/40b23475b15c38069e01c845aa70a58b2448113f

I believe that catches that and another mistake